gradio/pipelines.py

```diff
@@ -19,14 +19,17 @@ def load_from_pipeline(pipeline: pipelines.base.Pipeline) -> dict:
     (dict): a dictionary of kwargs that can be used to construct an Interface object
     """
     try:
-        import transformers
+        import transformers, diffusers
         from transformers import pipelines
+        from diffusers import DiffusionPipeline
+        from diffusers import pipelines as diffuser_pipelines
+
```

Comment:
Instead of having these within one block, which would mean that a user needs both `diffusers` and `transformers` installed, we should separate them so that only one is needed

Reply:
Added this